ui/src/features/get-test-reports.js

```diff
@@ -21,7 +21,7 @@ import DeleteDialog from "./components/DeleteDialog";
 const noDataMsg = 'There is no data to display.';
 const errorMsgGetReports = 'Error occurred while trying to get all reports for test.';
 const columnsNames = ['compare', 'start_time', 'end_time', 'duration', 'status', 'arrival_rate',
-    'ramp_to', 'last_success_rate', 'avg_rps', 'parallelism', 'notes', 'grafana_report', 'report', 'score', 'rerun', 'raw', 'logs'];
+    'ramp_to', 'last_success_rate', 'avg_rps', 'parallelism', 'notes', 'score', 'report', 'grafana_report', 'rerun', 'raw', 'logs'];
```

Comment:
something changed with the allignment.. please check it
both in last reports
and reports by test